### ISSUE TITLE
No need to install and trust cert on mac

### ISSUE THREAD

**Issue by DevinFu** [ProxymanApp/Proxyman#1778]
I'm a Mac user who previously relied on Charles as my proxy tool. However, I recently discovered Proxyman, an excellent alternative with a beautiful UI and powerful functions that I now prefer over Charles.

However, I have encountered an issue. Unlike with Charles, where I could proxy an Android device without the need to install and trust a certificate on my Mac, Proxyman requires me to follow your guide and go through this extra step. I believe this requirement is unnecessary and kindly request that you address it in a future version. Thank you for your great work.

**Comment by NghiaTranUIT** [comment]
Yes, you don't need to install the certificate on your Mac in order to capture & decrypt traffic from your android.

You can skip it. It's optional.

I will redesign this UI to make it clear.
